docs/api.rst

```diff
@@ -0,0 +1,7 @@
+API Documentation
+=================
+
+.. autosummary::
+    :toctree: autosummary
+
+    itctools.procedures.ITCProtocol
```

Comment:
Should we add more modules to the API autodocs?

Reply:
Definitely! Should I add all of them? where should I start?